<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="filterLine"/>
           </item>
           <item>
            <widget class="QToolButton" name="helpItems">
             <property name="text">
              <string/>
             </property>
             <property name="autoRaise">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="FlowItemTreeWidget" name="toolBox"/>
        </item>
       </layout>
      </widget>
      <widget class="FlowContainer" name="tabWidget"/>
     </widget>
     <widget class="ButtonBar" name="buttonBar" native="true"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ButtonBar</class>
   <extends>QWidget</extends>
   <header location="global">core/ButtonBar</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FlowContainer</class>
   <extends>QTabWidget</extends>
   <header location="global">core/FlowContainer</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FlowItemTreeWidget</class>
   <extends>QTreeView</extends>
   <header location="global">core/FlowItemTreeWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
